src/v2/styles/constants.ts

```diff
@@ -65,11 +69,31 @@ export const RADII = {
   subtle: '0.125em',
 }
 
+interface ConstantTypes {
+  emptySpaceWidth: string
+  doubleEmptySpaceWidth: string
+  headerHeight: string
+  blockWidth: string
+  blockGutter: string
+  doubleBlockGutter: string
+  blockAndGutter: string
+  containerOffset: string
+  blockPreviewWidth: string
+  legacyUnit: string
+  topBarHeight: string
+  // TODO give these real types
+  media: any
+  radii: any
+  breakpoints: any
+  space: any
+  z: any
+}
```

Comment:
You can just declare the theme in a const then `as const` and use `type Theme = typeof THEME` or whatever